core/domain/suggestion_services.py

```diff
@@ -1766,11 +1767,11 @@ def update_translation_suggestion(
     suggestion = get_suggestion_by_id(suggestion_id)
 
     # Clean the translation HTML if not a list of strings.
-    # Here, change is of type BaseChange and all attributes on BaseChange are
-    # created dynamically except cmd, so due this MyPy is unable to recognize
-    # `translation_html` as an attribute of change and throwing `"BaseChange"
-    # has no attribute "translation_html"` error. Thus to avoid the error, we
-    # used ignore here.
+    # Here we use MyPy ignore because change is of type BaseChange and all
```

Comment:
Same note about using subclasses.

Reply:
This issue is already fixed in https://github.com/oppia/oppia/pull/15958